app/src/test/kotlin/compiler/e2e/CorrectPrograms.kt

```diff
@@ -0,0 +1,182 @@
+package compiler.e2e
+
+import kotlin.test.Test
+
+class CorrectPrograms {
+    @Test
+    fun `test identifiers`() {
+        E2eAsserter.assertProgramCorrect(
+            """
+            zm n: Liczba = 10234
+            zm camelCaseIdentifier: Liczba = 10234
+            zm snake_case_identifier: Liczba = 10234
+            zm ident1f13r_w1tH_n00mb3r5: Liczba = 10234
+            """
+        )
+    }
+
+    @Test
+    fun `test types and literals`() {
+        E2eAsserter.assertProgramCorrect(
+            """
+            zm n: Liczba = 10234
+            n = 0
+            n = -0
+            n = -56789
+            n = +56789
+            n = 2147483647 // 32 bitowa ze znakiem
+            n = -2147483648
+
+            zm b: Czy = prawda
+            b = fałsz
```

Comment:
Global scope can only contain function or global variable declarations. Wrap this in a function.